cmd/notation/list.go

```diff
@@ -55,9 +59,26 @@ func runList(command *cobra.Command, opts *listOpts) error {
 	}
 
 	// write out
-	for _, sigManifest := range sigManifests {
-		fmt.Println(sigManifest.Blob.Digest)
+	return output(manifestDesc.Digest, sigManifests, reference)
+}
+
+func output(digest digest.Digest, sigManifests []notationRegistry.SignatureManifest, reference string) error {
+	ref, err := registry.ParseReference(reference)
+	if err != nil {
+		return err
 	}
 
+	sigCount := len(sigManifests)
+	if sigCount > 0 {
+		// print title
+		fmt.Printf("%s/%s@%s\n", ref.Registry, ref.Repository, digest)
```

Comment:
Try
```go
ref.Digest = digest
fmt.Println(ref)
```

Reply:
Tried, but doesn't work.
ref.Digest() is a method which parses ref.Reference to get digest.